### DISCUSSION TITLE
Received Version 1.29.0-dev instead of Version 1.28.0/.1

### DISCUSSION THREAD

**Discussion by TThommy** [teslamate-org/teslamate#3495] [Q&A] [answered]
Hello all,
thanks to the email notification I pulled the latest docker image (as always) and updated my teslamate docker instance. "unfortunately" I received version 1.29.0-dev instead of the last public built. I can (currently) live with it, but maybe some other users are also affected.
Best regards
Thomas

**Comment by cwanja** [comment, marked as answer]
Resolved with 26494df
